### DISCUSSION TITLE
Can you add custom models and skins or not?

### DISCUSSION THREAD

**Discussion by Draxtoze** [diamond-gold/MiniBosses#37] [Q&A] [answered]
Can you add custom models and skins or not? I would really be helpfull

**Comment by diamond-gold** [comment, marked as answer]
Yes you can. Do check out the [wiki](https://github.com/diamond-gold/MiniBosses/wiki)

**Reply by Draxtoze** [reply]
what network id should i put when i do custom mobs? a name i invent?

**Reply by diamond-gold** [reply]
If you are using skins, put `minecraft:player`, otherwise put the same as in your resource pack

**Reply by Draxtoze** [reply]
ok, so i can put for exemple custom:myboss , and put that network id?

**Reply by diamond-gold** [reply]
Yep thats right
